static/templates/group_pms.hbs

```diff
@@ -7,7 +7,7 @@
         {{else}}
         <span class="user_circle_fraction" style="background:none; border-color:hsl(0, 0%, 50%);"></span>
         {{/if}}
-        <a href="{{href}}" data-name="{{name}}" title="{{name}}" class="group-pm-link">{{short_name}}</a>
+        <a href="{{href}}" data-name="{{name}}" class="group-pm-link">{{short_name}}</a>
```

Comment:
I removed the tooltip for Group PMs on the right sidebar, as suggested, but I've also removed the HTML title that we originally had. Should I leave it here, or is this correct?

Reply:
@rishig